File: app/src/main/java/com/example/weatherchallengemvvmkotlin/data/models/WeatherObject.java
package com.example.weatherchallengemvvmkotlin.data.models;

import com.google.gson.annotations.SerializedName;

public class WeatherObject {

    @SerializedName("id")
    int id;

    @SerializedName("main")
    String main;

    @SerializedName("description")
    String description;

    @SerializedName("icon")
    String icon;

    public int getId() {
        return id;
    }

    public String getMain() {
        return main;
    }

    public String getDescription() {
        return description;
    }

    public String getIcon() {
        return icon;
    }
}

File: app/src/main/java/com/example/weatherchallengemvvmkotlin/data/models/WeatherObject.kt
package com.example.weatherchallengemvvmkotlin.data.models

import com.google.gson.annotations.SerializedName

class WeatherObject {
    @SerializedName("id") var id : Int = 0
    @SerializedName("main") var main : String? = null
    @SerializedName("description") var description : String? = null
    @SerializedName("icon") var icon : String? = null
}